hoomd/mpcd/module.cc

```diff
@@ -34,6 +34,7 @@ void export_ManualVirtualParticleFiller(pybind11::module&);
 void export_NoForce(pybind11::module&);
 void export_ParallelPlateGeometry(pybind11::module&);
 void export_ParallelPlateGeometryFiller(pybind11::module&);
+void export_ConcentricCylindersGeometry(pybind11::module&);
```

Comment:
Make sure to add the filler (here and below for the GPU), then call those exports. The filler will be automatically created using the rejection sampling method.